apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -127,56 +124,9 @@ class SectionProgress extends Component {
       },
       {includeUserId: true}
     );
-  };
```

Comment:
tooltips are now rendered in their respective components, e.g. `ProgressTableLessonNumber` and `ProgressTableStudentList`